lib/acme/client.rb

```diff
@@ -43,6 +43,7 @@ def initialize(jwk: nil, kid: nil, private_key: nil, directory: DEFAULT_DIRECTOR
     @kid, @connection_options = kid, connection_options
     @bad_nonce_retry = bad_nonce_retry
     @directory = Acme::Client::Resources::Directory.new(URI(directory), @connection_options)
+    @alternate_links ||= []
```

Comment:
oh, whoops this is my mistake. I don't think you want to store these on the Client. Perhaps the `certificate` method on this class should return a tuple of `[response.body, alternate_links]` so that they can be stored on the Order class.

Reply:
I understand. I'll have to set the save destination to the Order class, or there will be a side effect.